lib/rsk-utils.js

```diff
@@ -559,6 +567,33 @@ const waitForRskMempoolToGetAtLeastThisManyTxs = async (rskTxHelper, atLeastExpe
   throw new Error(`[${waitForRskMempoolToGetAtLeastThisManyTxs.name}] rsk mempool didn't get at least ${atLeastExpectedCount} txs after ${maxAttempts} attempts`);
 };
 
+const uncompressPublicKey = (compressedPublicKey) => {
+  const uncompressedPublicKey = secp.ProjectivePoint.fromHex(removePrefix0x(compressedPublicKey));
+  return uncompressedPublicKey.toHex(false);
+};
+
+const keccak256 = (str) => {
+  return Web3.utils.keccak256(str);
+}
+
+const removeCompressionPrefix = (uncompressedPublicKey) => {
+  if(uncompressedPublicKey.startsWith('04')) {
+      return uncompressedPublicKey.substring(2);
+  } else if(uncompressedPublicKey.startsWith('0x04')) {
+      return uncompressedPublicKey.substring(4);
+  }
```

Comment:
```suggestion
  if (uncompressedPublicKey.startsWith('04')) {
      return uncompressedPublicKey.substring(2);
  }
  if (uncompressedPublicKey.startsWith('0x04')) {
      return uncompressedPublicKey.substring(4);
  }
```

no need for the `else`, since you are returning right?

Reply:
Yes, and now.
With the current logic, what was passed will be returned intact if no condition is matched.
If we remove that `else if`, it will then remove the first 4 characters of the passed string no matter what.

Reply:
🤔